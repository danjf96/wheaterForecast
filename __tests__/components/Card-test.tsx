/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Card from '../../src/components/card'
 // Note: test renderer must be required after react-native.
 import { render } from 'react-native-testing-library';
import { Text } from 'react-native';
 
 it('renders correctly', async () => {
   const screen = render(<Card />)
   expect(screen.toJSON()).toMatchSnapshot()
 });

 it('renders children', async () => {
  const screen = render(<Card> 
    <Text>CHILDREN</Text>
  </Card>)
  expect(screen.toJSON()).toMatchSnapshot()
  expect(screen.getByText('CHILDREN')).toBeTruthy()
});

 