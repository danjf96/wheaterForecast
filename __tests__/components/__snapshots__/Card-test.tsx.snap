// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#00AAFF",
        "borderColor": "#0086ff",
        "borderRadius": 15,
        "borderWidth": 1,
        "elevation": 1,
        "marginTop": 25,
        "paddingHorizontal": 20,
        "paddingVertical": 15,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <Text>
    CHILDREN
  </Text>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#00AAFF",
        "borderColor": "#0086ff",
        "borderRadius": 15,
        "borderWidth": 1,
        "elevation": 1,
        "marginTop": 25,
        "paddingHorizontal": 20,
        "paddingVertical": 15,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
